src/credentials-manager/index.js

```diff
@@ -0,0 +1,153 @@
+import {Platform, NativeModules} from 'react-native';
+import CredentialsManagerError from './credentialsManagerError';
+
+export default class CredentialsManager {
+  /**
+   * Construct an instance of CredentialsManager which can be used to
+   * store, retrieve and manage credentials.
+   *
+   * @param {*} auth - required - instance of Auth0 Auth API
+   */
```

Comment:
I think this is an improvement but we should document the actual fields like we've done for other methods (`saveCredentials`, for example).

> instance of Auth0 Auth API

Is this correct? Why do we require the whole instance of the Auth API, only to extract `domain` and `clientId` and then not use the instance?